Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/CancelRide/Internal.hs

```diff
@@ -146,20 +140,16 @@ cancelRideImpl rideId bookingCReason = do
 cancelRideTransaction ::
   ( EsqDBFlow m r,
     CacheFlow m r,
-    Esq.EsqDBReplicaFlow m r,
-    EsqLocDBFlow m r,
-    EsqLocRepDBFlow m r,
-    HasField "enableLocationTrackingService" r Bool
+    Esq.EsqDBReplicaFlow m r
   ) =>
   Id SRB.Booking ->
   DRide.Ride ->
   SBCR.BookingCancellationReason ->
   Id DMerc.Merchant ->
   m ()
-cancelRideTransaction bookingId ride bookingCReason merchantId = do
+cancelRideTransaction bookingId ride bookingCReason _merchantId = do
   let driverId = cast ride.driverId
   driverInfo <- QDI.findById (cast ride.driverId) >>= fromMaybeM (PersonNotFound ride.driverId.getId)
-  void $ DLoc.updateOnRide merchantId ride.driverId False
```

Comment:
@chakri642 Here Also we should call our Ride Cancelled API if not already handled by our common CancelRideImpl function.